app/src/main/java/kr/co/nottodo/data/remote/api/ResponseRecommendationCategoryListService.kt

```diff
@@ -0,0 +1,12 @@
+package kr.co.nottodo.data.remote.api
+
+import kr.co.nottodo.presentation.toplevel.recommendation.data.responsedto.ResponseRecommendationCategoryListDto
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface RecommendationCategoryListService {
+    @GET("environment/{Path}")
+    fun getCategoryList(@Path("id") id: Int): Call<ResponseRecommendationCategoryListDto>
```

Comment:
```suggestion
    @GET("environment/{id}")
    fun getCategoryList(@Path("id") id: Int): Call<ResponseRecommendationCategoryListDto>
```

Path 어노테이션(골뱅이) 안에 있는 문자열값과 @GET 안에 있는 중괄호 안에 있는 문자열은 같아야 서로 연결이 돼...!!